assets/sass/_pulumi-up.scss

```diff
@@ -0,0 +1,100 @@
+// Neon Animation found on Codepen
+// https://codepen.io/Trinca/pen/NAvpWa
+$pageBackgroundColor: #11151c;
```

Comment:
Not a huge deal, but why this color and not one from the palette -- say, `bg-gray-900`, which looks pretty similar?

Reply:
👍